src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,19 @@
+@keyframes moveSec {
+  0% {
+    transform: #{$second-translate} rotate(0);
+  }
+
+  100% {
+    transform: #{$second-translate} rotate(360deg);
+  }
+}
```

Comment:
I think that it is possible just use rotate and transform-origin property. I am not sure, but I think you can create just one animation with propertu "rotate: and use different duration in different hands

Reply:
yeah, it's possible to make this animation only with rotate